README.md

```diff
@@ -37,12 +38,35 @@ regenerate Go code after you change the `sourcegraph.proto` file.
    git checkout proto3
    go install ./...
    ```
-1. **Install [gen-mocks](https://sourcegraph.com/sourcegraph/gen-mocks)** by running:
+
+3. **Install [gen-mocks](https://sourcegraph.com/sourcegraph/gen-mocks)** by running:
 
    ```
    go get -u sourcegraph.com/sourcegraph/gen-mocks
    ```
 
+4. **Install `gopathexec`**:
+
+   ```
+   go get -u github.com/shurcooL/gopathexec
+   ```
+
+#### OS X-specific
+
+```
+brew install gnu-sed --with-default-names
```

Comment:
what about making the go:generate line call `sh -c` and use simple shell scripting to switch between OS X and Linux? (checking `uname` or something.) Or something like that.